2023-06-28 16:31:26,900 Job "Lvl2Trader.update_all_thresholds (trigger: cron[day_of_week='mon-fri', hour='20', minute='30'], next run at: 2023-06-29 20:30:00 UTC)" raised an exception
Traceback (most recent call last):
  File "/Users/mattjzhou/opt/miniconda3/envs/trading_env/lib/python3.11/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mattjzhou/riverhill/lvl2trader.py", line 133, in update_all_thresholds
    prev_long_row, prev_short_row = self.test_thresholds(symbol, hour, \
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mattjzhou/riverhill/lvl2trader.py", line 121, in test_thresholds
    return long_df[(long_df["Buy Threshold"]==buy_threshold) & (long_df["Sell Threshold"]==sell_threshold)].iloc[0].tolist(), \
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^
  File "/Users/mattjzhou/opt/miniconda3/envs/trading_env/lib/python3.11/site-packages/pandas/core/indexing.py", line 1103, in __getitem__
    return self._getitem_axis(maybe_callable, axis=axis)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mattjzhou/opt/miniconda3/envs/trading_env/lib/python3.11/site-packages/pandas/core/indexing.py", line 1656, in _getitem_axis
    self._validate_integer(key, axis)
  File "/Users/mattjzhou/opt/miniconda3/envs/trading_env/lib/python3.11/site-packages/pandas/core/indexing.py", line 1589, in _validate_integer
    raise IndexError("single positional indexer is out-of-bounds")
IndexError: single positional indexer is out-of-bounds
